srtcore/filelist.maf

```diff
@@ -52,4 +52,3 @@ srt_compat.h
 threadname.h
 utilities.h
 window.h
-
```

Comment:
New line is wrongfully deleted here.

Reply:
what do you mean? the file now looks identical to the same file in master - i don't know why the compare tool in github denies there is a trailing newline - but it's there and i didn't delete it. It could be a windows / linux CR/CRLF funny - but the github site itself shows that line 55 exists still in this PR.

Do we need to worry that much given that the plan to append in the next PR 4 extra lines anyway?? I can promise to terminate with a double-newline if that helps then... i just want this all to be over...